.github/workflows/molecule.yml

```diff
@@ -0,0 +1,30 @@
+name: Test with Molecule
+on:
+  workflow_call:
+    inputs:
+      run_tags:
+        required: false
+        type: string
+        default: all
+
+jobs:
+  molecule:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        scenario:
+          - centos7
```

Comment:
Only `centos7`?

Reply:
yeah for now, I'll add Rocky 9 in a separate pr